webpack/assets/javascripts/react_app/components/charts/StatisticsChartsListStyles.js

```diff
@@ -0,0 +1,27 @@
+export default {
```

Comment:
This approach was spurred on by facebook software engineer, Chris Chedeau, in his [React: CSS in JS](https://speakerdeck.com/vjeux/react-css-in-js) identified 7 problems with CSS at scale, including global namespace, no explicit dependencies, sharing constants, non-deterministic resolution and isolation. These problems can be solved relatively easily by writing styles in JS.

Slides 1-24 of the above referenced presentation describe these problems and show some hacks that facebook used to get around them. Starting from Slide 25, he shows how to solve the same problems elegantly with JS inline styles.

I've looked at a couple of strategies for styling React components and at the moment inline styles seems the simplest and most readable solution.

I think that this approach also will facilitate extracting encapsulated components for reuse and possible combination in libraries.

As we proceed we make wish to re-examine this strategy in a separate PR.